client/canary/scripts/gitServer/install.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+echo "==== Deploying Gogs Git server with custom certificate ===="
+
+# Find the directory we're in (used to reference other scripts)
+my_dir=$(dirname $(readlink -f $0))
+# The main directory of canary-scripts
+root_dir=$my_dir/../../../..
+kubeconfig=/opt/e2e/default-kubeconfigs/hub
+
+KUBECTL_CMD="oc --kubeconfig /opt/e2e/default-kubeconfigs/hub"
```

Comment:
should it be "kubectl -- kubeconfig blabla" as only kubectl is installed in the app lifecycle e2e backend container

Reply:
@xiangjingli In the scripts/utils directory, I have a script to install `oc` because I need it to create an OCP route which `kubectl` does not support.

Reply:
@xiangjingli I removed scripts/utils/download_tools.sh script and updated the Dockerfile to install `oc`.